app/src/main/java/com/plaid/linksample/MainActivityStartActivityForResult.kt

```diff
@@ -4,60 +4,47 @@
 
```

Comment:
This is the previous `MainActivity`, without the options menu to switch to other examples.